lib/mobility-core/src/Kernel/External/Verification/Interface/Types.hs

```diff
@@ -21,9 +21,10 @@ where
 import Deriving.Aeson
 import qualified Kernel.External.Verification.Idfy.Config as Idfy
 import qualified Kernel.External.Verification.Idfy.Types.Response as Idfy
+import qualified Kernel.External.Verification.InternalScripts.Types as FV
 import Kernel.Prelude
 
-newtype VerificationServiceConfig = IdfyConfig Idfy.IdfyCfg
+data VerificationServiceConfig = IdfyConfig Idfy.IdfyCfg | FaceVerificationConfig FV.FaceVerificationCfg
```

Comment:
i think we should treat these as separate services ? One for DL/RC and other for Facial ? seems mutually exclusive to me.

Reply:
@hkmangla asked it to be added as a part of the verification service.